frontend/src/components/DefenceBox/DefenceConfigurationRadioButton.css

```diff
@@ -28,8 +31,8 @@
 .defence-radio-button input + span::after {
 	content: '';
 	position: absolute;
-	top: 0.3125rem;
-	left: 0.3125rem;
+	top: 0.34rem;
+	left: 0.34rem;
```

Comment:
This change has caused the little "pupil" of the bottom two radio buttons to be off-centre. What was the intention of the change?

![image](https://github.com/ScottLogic/prompt-injection/assets/118171430/fc310851-79e2-42ea-a8f7-14651f1c73d1)

Reply:
the original had the button off centered (if you look at the before picture)

Reply:
weird. For me it's on centre on my machine (big 1080p monitor, full screen, no zoom), and then off-centre after the change

Reply:
well its not in the scope of this ticket so i'll just revert it

Reply:
I think we can solve this with a simpler css structure, but that's for another day. I've seen perfectly good radiobuttons with custom styling that only have a `::before` pseudo-element, not both before and after, which is what seems to be throwing alignment off here.